website/snippets/_privatelink-across-providers.md

```diff
@@ -0,0 +1 @@
+PrivateLink endpoints can't connect across cloud providers. For a PrivateLink connection to work, both dbt Cloud and the server (like {props.type}) must be hosted on the same cloud provider. For example, AWS PrivateLink can’t connect to resources hosted on Azure, and Azure PrivateLink can’t connect to resources hosted on AWS.
```

Comment:
do we want to mention what alternatives to consider if users _are_ in this situation? e.g. do they need to consider alternatives like a VPN or any other solution?